source/cluster_manager/matcher.js

```diff
@@ -124,7 +173,31 @@ var generalMatcher = function () {
     };
 };
 
-var audioMatcher = generalMatcher;
+var audioMatcher = function () {
```

Comment:
Is this same as `webrtcMatcher`?
What I recommend is that reusing the previous function instead of copy-paste code.